src/citrine/_rest/resource.py

```diff
@@ -9,14 +31,11 @@ class Resource(Serializable[Self]):
 
     _path_template: str = NotImplemented
     _response_key: Optional[str] = None
-
-    def resource_type(self) -> str:
```

Comment:
I feel OK deleting this method because it's on an internal class.

Not sure if this merits a minor version bump or if it's a patch because the change is entirely internal.

Reply:
I'd go ahead and use a minor bump since it wasn't `_protected`.